GeosCore/fullchem_mod.F90

```diff
@@ -595,16 +604,18 @@ SUBROUTINE Do_FullChem( Input_Opt,  State_Chm, State_Diag,                 &
        LWC       = 0.0_fp                   ! Liquid water content
        SIZE_RES  = .FALSE.                  ! Size resolved calculation?
        C         = 0.0_dp                   ! KPP species conc's
-       VAR       = 0.0_dp                   ! KPP variable species conc's
-       FIX       = 0.0_dp                   ! KPP fixed species conc's
        RCONST    = 0.0_dp                   ! KPP rate constants
        PHOTOL    = 0.0_dp                   ! Photolysis array for KPP
        K_CLD     = 0.0_dp                   ! Sulfur in-cloud rxn het rates
        K_MT      = 0.0_dp                   ! Sulfur sea salt rxn het rates
+       CFACTOR   = 1.0_dp                   ! KPP conversion factor
```

Comment:
We now set CFACTOR = 1 directly.  This had been done in Init_KPP() but this routine only sets C= 0 and CFACTOR=1, so it's kind of useless.  We can now do this in the body of the main parallel loop.